components/ui/table.tsx

```diff
@@ -88,9 +92,9 @@ const TableHead = React.forwardRef<
   <th
     ref={ref}
     className={cn(
-      'h-12 text-left align-middle font-serif font-normal text-primary-700 [&:has([role=checkbox])]:pr-0',
+      'h-12 min-w-16 text-left align-middle font-serif text-primary-700 [&:has([role=checkbox])]:pr-0',
       'px-1 py-2 sm:px-2 md:px-3 lg:px-4',
-      'text-sm sm:text-base md:text-lg',
```

Comment:
odd?

Reply:
Sorry, I was just experimenting with sizes. Didn't actually mean to commit that but still due to the font the heading look smaller than the text inside the tables